sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -172,12 +171,7 @@ def __init__(self):
     self._computed_pcolls = set()
     # Always watch __main__ module.
     self.watch('__main__')
-    # Do a warning level logging if current python version is below 3.6.
-    if sys.version_info < (3, 6):
-      self._is_py_version_ready = False
-      _LOGGER.warning('Interactive Beam requires Python 3.5.3+.')
-    else:
-      self._is_py_version_ready = True
+    self._is_py_version_ready = True
```

Comment:
We can probably let go of this variable in another change.

Reply:
It seems to be true, I'll make another PR.